x/interchainstaking/keeper/callbacks.go

```diff
@@ -253,6 +255,101 @@ func DepositIntervalCallback(k *Keeper, ctx sdk.Context, args []byte, query icqt
 	return nil
 }
 
+func ValidatorSigningInfosCallback(k *Keeper, ctx sdk.Context, args []byte, query icqtypes.Query) error {
+	zone, found := k.GetZone(ctx, query.GetChainId())
+	if !found {
+		return fmt.Errorf("no registered zone for chain id: %s", zone.ChainId)
+	}
+
+	k.Logger(ctx).Debug("Validator signing info callback", "zone", zone.ChainId)
+
+	signingInfoRes := slashingtypes.QuerySigningInfoResponse{}
```

Comment:
If we use the response here, there is no proof and the result can be forged. We can only update state based upon kv Store lookups